### PR TITLE
texturepacker cosmetics

### PR DESCRIPTION
**Pull request by stefansaraev**
as title says.

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
Ok for me
@ronie

**Comment by stefansaraev** [comment]
what you, guys, think about always using -use_none (no compressions) ?

**Comment by ronie** [comment]
dupecheck changes are certainly ok.

i have no idea whether compression is useful or not.

**Comment by anaconda** [comment]
I was wondering if `-dupecheck` is actually needed at all. I admit I'm not skinner, so I don't really know, but why would someone want duplicates in the XBT?
If it's needed for debugging skins or anyways it is useful for skinners, I'd propose to invert the default (remove `-dupecheck` and add `-no_dupecheck`), otherwise I think it can be removed and it enabled by default.

**Comment by MartijnKaijser** [comment]
@anaconda no we leave it as is

**Comment by stefansaraev** [comment]
-use_none -dupecheck: Textures.xbt size: 4995774 bytes
-use_lzo -dupecheck: Textures.xbt size: 4946675 bytes (same as -use_dxt)

so. compression seems not to make too much difference, just some overhead in runtime decompressing ?

File: tools/depends/native/TexturePacker/src/TexturePacker.cpp [hunk 1]
```diff
@@ -286,7 +286,7 @@ void Usage()
   puts("  -help            Show this screen.");
   puts("  -input <dir>     Input directory. Default: current dir");
   puts("  -output <dir>    Output directory/filename. Default: Textures.xpr");
-  puts("  -dupecheck       Enable duplicate file detection. Reduces output file size. Default: on");
+  puts("  -dupecheck       Enable duplicate file detection. Reduces output file size. Default: off");
```
Comment (wsnipex) [inline tools/depends/native/TexturePacker/src/TexturePacker.cpp]: in this case you should set dupecheck to false on L458

Comment (Montellese) [inline tools/depends/native/TexturePacker/src/TexturePacker.cpp]: Nope, the `dupecheck` variable is `false` by default and only set to `true` if one of the arguments matches `-dupecheck` (yes the syntax with `!strcmp()` looks confusing).

Comment (stefansaraev) [inline tools/depends/native/TexturePacker/src/TexturePacker.cpp]: nope. L458 sets dupecheck internaly if -dupecheck is used. default is already false at L426

Comment (wsnipex) [inline tools/depends/native/TexturePacker/src/TexturePacker.cpp]: ok, did look properly

**Comment by ronie** [comment]
ok to merge?

**Comment by mkortstiege** [comment]
No objections.

**Pull request merged**